lib/services/workflow-api-service.js

```diff
@@ -237,7 +237,39 @@ function workflowApiServiceFactory(
     WorkflowApiService.prototype.getWorkflowsByNodeId = function(id, query) {
         var nodeId = ({node: id});
         var mergedQuery = _.merge({}, nodeId, query);
-        return waterline.graphobjects.find(mergedQuery);
+        return waterline.graphobjects.find(mergedQuery).then(
+            function sanitize(workflow) {
```

Comment:
I'm open to a refactor in either direction... besides... this current commit has the code in the wrong spot anyway... sorry :( my next push with correct that...